packages/did/src/Did.chain.ts

```diff
@@ -246,16 +364,34 @@ export async function getAddKeyExtrinsic(
   )
 }
 
-export async function getDeleteDidExtrinsic(): Promise<Extrinsic> {
+export async function getAddEndpointExtrinsic(
+  endpoint: IDidServiceEndpoint
+): Promise<Extrinsic> {
+  const { api } = await BlockchainApiConnection.getConnectionOrConnect()
+  const encoded = encodeServiceEndpoint(api.registry, endpoint)
+  return api.tx.did.addServiceEndpoint(encoded)
+}
+
+export async function getRemoveEndpointExtrinsic(
+  endpointId: IDidServiceEndpoint['id']
+): Promise<Extrinsic> {
+  const { api } = await BlockchainApiConnection.getConnectionOrConnect()
+  return api.tx.did.removeServiceEndpoint(endpointId)
+}
+
+export async function getDeleteDidExtrinsic(
+  endpointsCount: number
+): Promise<Extrinsic> {
   const { api } = await BlockchainApiConnection.getConnectionOrConnect()
-  return api.tx.did.delete()
+  return api.tx.did.delete(endpointsCount)
 }
 
 export async function getReclaimDepositExtrinsic(
```

Comment:
Shouldn't this have a different naming convention, since the other `get` are returning an `Extrinsic`, however, `getReclaimDepositExtrinsic` is returning a `SubmittableExtrinsic`. 

How about `reclaimDepositExtrinsic`?

Reply:
We had a short discussion with Timo in a previous PR about the same subject -> https://github.com/KILTprotocol/sdk-js/pull/421#discussion_r730951935

Reply:
Will be handled in new ticket: https://github.com/KILTprotocol/ticket/issues/1675